docs/wiki/installation/cli-flags.md

```diff
@@ -389,45 +389,29 @@ The minimum level for status logs written to stderr. Use the following values: `
 
 The default behavior is to also write status logs to stderr. Set this flag to false to disable writing (copying) status logs to stderr. In this case `--verbose` is respected.
 
-`--host_identifier=hostname`
-
-Field used to identify the host running osquery: `hostname`, `uuid`, `ephemeral`, `instance`, `specified`.
-
-DHCP may assign variable hostnames, if this is the case, you may need a consistent logging label. Four options are available to you:
-
-- `uuid` uses the platform (DMTF) host UUID, fetched at process start.
-- `instance` uses an instance-unique UUID generated at process start, persisted in the backing store.
-- `ephemeral` uses an instance-unique UUID generated at process start, not persisted.
-- `specified` uses an ID provided by the `--specified_identifier` flag.
-
-`--specified_identifier=this.is.the.identifier`
-
-If `--host_identifier=specified` is set, use this value as the host identifier.
+`--logger_path=/var/log/osquery/`
 
-`--verbose=false`
+Directory path for `ERROR`/`WARN`/`INFO` and query result logging by the **filesystem** plugin.
 
-Enable verbose informational messages.
+`--logger_mode=420`
 
-`--thrift_verbose=false`
+File mode for output log files by the **filesystem** plugin (provided as a decimal string). Note that this affects both the query result log and the status logs. **Warning**: If run as root, log files may contain sensitive information!
```

Comment:
Ah, I only moved this from another place in the docs. If we need to change this then we should do it in another PR.

Reply:
Ha. I'm sure that you'd encourage me to clean it up while here. Either way though

Reply:
Is the documentation incorrect?

Reply:
Docs are incorrect.

Unix permissions are octal. They're often described as decimal, since if you're not doing bit manipulation it doesn't really matter -- `7` = `111` = `rwx`.

But if you view them as an aggregate,  you need octal.  `744` =  `111100100` = `1111 001 001`